features/nanostack/mbed-mesh-api/source/wisun_tasklet.c

```diff
@@ -210,6 +213,18 @@ static void wisun_tasklet_configure_and_connect_to_network(void)
                             network_name,
                             fhss_timer_ptr);
 
+#if defined(MBED_CONF_MBED_MESH_API_CERTIFICATE_HEADER)
+    arm_certificate_chain_entry_s chain_info;
+    memset(&chain_info, 0, sizeof(arm_certificate_chain_entry_s));
+    chain_info.cert_chain[0] = (const uint8_t *) MBED_CONF_MBED_MESH_API_ROOT_CERTIFICATE;
+    chain_info.cert_len[0] = strlen((const char *) MBED_CONF_MBED_MESH_API_ROOT_CERTIFICATE) + 1;
```

Comment:
Could you check configuration validity at compile time to avoid hard fault caused by illegal configuration?

Reply:
Check added